### DISCUSSION TITLE
Disable hover cards?

### DISCUSSION THREAD

**Discussion by timholy** [community/community#45419] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Recently, developer hover cards seem to have appeared or gotten more prominent---for example, if I accidentally drag my mouse over my GitHub handle while heading for the `Commits` tab, I get this:

![image](https://user-images.githubusercontent.com/1525481/214818304-133649b8-3816-4d6c-9a47-5f4b50281703.png)

Obviously, to click on the `Commits` tab I need to get rid of the hover card.

However, the card is rather persistent: waiting doesn't dismiss it, moving the mouse doesn't dismiss it, nor does scrolling the window. The only thing I've found to get rid of it is to find something *else* I can hover over that puts up a hover card in a place where it won't get in my way. It's pretty irritating because they get in my way and offer negligible benefit.

I'm on Firefox 109.0 64-bit windows.

Can I turn off the hover cards?

**Comment by VladCuciureanu** [comment, marked as answer]
So, ignoring the spammy reply, this is an issue that is known by GitHub and is working on it
https://github.com/orgs/community/discussions/44630
